src/main/java/frc/robot/subsystems/Scoring.java

```diff
@@ -21,10 +21,7 @@ public class Scoring extends PIDSubsystem {
             ScoringConstants.rightScoringElevatorMotorPort, MotorType.kBrushless);
     private final CANSparkMax rollerMotor = new CANSparkMax(
             ScoringConstants.rollerMotorPort, MotorType.kBrushless);
-    private final CANSparkMax leftLoadingMotor = new CANSparkMax(
-            ScoringConstants.leftScoringElevatorMotorPort, MotorType.kBrushless);
-    private final CANSparkMax rightLoadingMotor = new CANSparkMax(
-            ScoringConstants.rightScoringElevatorMotorPort, MotorType.kBrushless);
+
     private AbsoluteEncoder leftScoringElevatorEncoder = leftScoringElevatorMotor.getAbsoluteEncoder(Type.kDutyCycle);
 
     public enum ElevatorSetpoints {
```

Comment:
For the enum, it is standard practice to fully capitalize the names of the constants.

Reply:
I'm not a fan of fully capitalized enums, but I am a fan of `PascalCase` enums 
(sidenote: `PascalCase`, `camelCase`, `snake_case`, `UPPER_SNAKE_CASE`)